{"name": "test_login_valid", "status": "failed", "statusDetails": {"message": "AssertionError: Locator expected to have text 'test1161'\nActual value: test1111 \nCall log:\n  - Expect \"to_have_text\" with timeout 5000ms\n  - waiting for locator(\".username\")\n    9 × locator resolved to <span class=\"username\">test1111</span>\n      - unexpected value \"test1111\"", "trace": "page = <Page url='https://hocvancokimngan.com/'>, test_users = {'editor': {'email': 'test1111@gmail.com', 'password': '123456'}, 'viewer': {'email': 'viewer@example.com', 'password': 'P@ssw0rd'}}\n\n    @pytest.mark.ui\n    @pytest.mark.auth\n    @pytest.mark.smoke\n    @allure.feature(\"Auth\")\n    @allure.story(\"Login\")\n    def test_login_valid(page, test_users):\n        AuthPage(page).goto_login().login(\n            test_users[\"editor\"][\"email\"],\n            test_users[\"editor\"][\"password\"]\n>       ).assert_logged_in(\"test1161\")\n\ntests\\auth\\test_login_valid.py:14: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pages.auth_page.AuthPage object at 0x0000024DE35ACE00>, username = 'test1161'\n\n    def assert_logged_in(self, username):\n        with allure.step(\"Xác minh đã đăng nhập thành công\"):\n>           expect(self.get_by_class(self.lbl_username)).to_have_text(username)\nE           AssertionError: Locator expected to have text 'test1161'\nE           Actual value: test1111 \nE           Call log:\nE             - Expect \"to_have_text\" with timeout 5000ms\nE             - waiting for locator(\".username\")\nE               9 × locator resolved to <span class=\"username\">test1111</span>\nE                 - unexpected value \"test1111\"\n\npages\\auth_page.py:45: AssertionError"}, "steps": [{"name": "Goto Login", "status": "passed", "start": 1756046125379, "stop": 1756046126356}, {"name": "Nhập thông tin đăng nhập", "status": "passed", "start": 1756046126356, "stop": 1756046126496}, {"name": "Đăng nhập", "status": "passed", "start": 1756046126496, "stop": 1756046127392}, {"name": "Xác minh đã đăng nhập thành công", "status": "failed", "statusDetails": {"message": "AssertionError: Locator expected to have text 'test1161'\nActual value: test1111 \nCall log:\n  - Expect \"to_have_text\" with timeout 5000ms\n  - waiting for locator(\".username\")\n    9 × locator resolved to <span class=\"username\">test1111</span>\n      - unexpected value \"test1111\"\n\n", "trace": "  File \"C:\\Users\\thanh\\PycharmProjects\\demo_pytest_playwright_web\\pages\\auth_page.py\", line 45, in assert_logged_in\n    expect(self.get_by_class(self.lbl_username)).to_have_text(username)\n  File \"C:\\Users\\thanh\\PycharmProjects\\demo_pytest_playwright_web\\.venv\\Lib\\site-packages\\playwright\\sync_api\\_generated.py\", line 19958, in to_have_text\n    self._sync(\n  File \"C:\\Users\\thanh\\PycharmProjects\\demo_pytest_playwright_web\\.venv\\Lib\\site-packages\\playwright\\_impl\\_sync_base.py\", line 115, in _sync\n    return task.result()\n           ^^^^^^^^^^^^^\n  File \"C:\\Users\\thanh\\PycharmProjects\\demo_pytest_playwright_web\\.venv\\Lib\\site-packages\\playwright\\_impl\\_assertions.py\", line 559, in to_have_text\n    await self._expect_impl(\n  File \"C:\\Users\\thanh\\PycharmProjects\\demo_pytest_playwright_web\\.venv\\Lib\\site-packages\\playwright\\_impl\\_assertions.py\", line 83, in _expect_impl\n    raise AssertionError(\n"}, "start": 1756046127392, "stop": 1756046132401}], "start": 1756046125379, "stop": 1756046132401, "uuid": "be407d8d-8a9c-41f2-82e4-699d13f86ca8", "historyId": "f370b6ae400907e33b53daf29861cd3c", "testCaseId": "f370b6ae400907e33b53daf29861cd3c", "fullName": "tests.auth.test_login_valid#test_login_valid", "labels": [{"name": "feature", "value": "Auth"}, {"name": "story", "value": "Login"}, {"name": "tag", "value": "smoke"}, {"name": "tag", "value": "auth"}, {"name": "tag", "value": "ui"}, {"name": "parentSuite", "value": "tests.auth"}, {"name": "suite", "value": "test_login_valid"}, {"name": "host", "value": "Thanh-PC"}, {"name": "thread", "value": "23236-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.auth.test_login_valid"}]}